### DISCUSSION TITLE
Lesson 6: How to delete a network from brownie

### DISCUSSION THREAD

**Discussion by ace-brown** [smartcontractkit/full-blockchain-solidity-course-py#237] [Q&A] [answered]
Hi everyone. At lesson 6 around hour 5:38, I added the ganache-local to the network list. However, I entered the http of the host address with capital letter like this **HTTP**://127.0.0.1:7545, while based on the video this format (**http**://127.0.0.1:7545) is correct. Now, how can I delete that network? Or how can i edit it, if possible?

**Comment by cromewar** [comment, marked as answer]
As the command says:
![image](https://user-images.githubusercontent.com/20199180/138285919-94efdef9-5b61-44dd-a2b7-f15f67759186.png)

So, using `brownie networks list` the id is the word on green

![image](https://user-images.githubusercontent.com/20199180/138286206-f8332450-d699-4d0c-9dc7-6c9110a129f4.png)

let's suppose I want to modify mainnet-fork network

`brownie networks modify mainnet-fork host=https://your_url_here`

**Comment by ace-brown** [comment]
so where should i insert the command? for ex, like this:

brownie networks modify mainnet-fork host=https://your_url_here delete mainnet-fork
**or** 
delete mainnet-fork
**then** 
brownie networks modify mainnet-fork host=https://your_url_here

**Reply by cromewar** [reply]
if you want to delete it entirely use 

`brownie networks delete IdOfTheNet`

If you want to just modify a parameter

`bronwie networks modify  IdOfTheNet param=`

**Reply by ace-brown** [reply]
Thank you again for your time and efforts.
